aslam_offline_calibration/kalibr/launch/undistort_kalibr.launch

```diff
@@ -0,0 +1,111 @@
+<?xml version="1.0"?>
+
+<launch>
+  <!-- This argument needs to be set for importing the params. -->
```

Comment:
optional: you can add `doc` attributes to the `arg` elements, and put the comments/description there.